src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Calculator implements Calculable {
+
+    @Override
+    public double calculate(int operand1, int operand2, char operator) {
+        switch (operator) {
+            case '+':
+                return (double) operand1 + (double) operand2;
```

Comment:
```suggestion
                return (double) operand1 + operand2;
```
same for other operations - this will be enough